clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -692,23 +692,22 @@ static RValue emitLibraryCall(CodeGenFunction &CGF, const FunctionDecl *FD,
   RValue Call =
       CGF.EmitCall(E->getCallee()->getType(), callee, E, ReturnValueSlot());
 
-  // Check the supported intrinsic.
+  ASTContext &Context = CGF.getContext();
   if (unsigned BuiltinID = FD->getBuiltinID()) {
     auto IsErrnoIntrinsic = [&]() -> unsigned {
```

Comment:
The name here IsErrnoIntrinsic is incorrect for fabs which cannot set errno

Reply:
Oh, your are right, so maybe list them one by one ?